geode-core/src/distributedTest/java/org/apache/geode/internal/cache/BrokenSerializationConsistencyRegressionTest.java

```diff
@@ -15,8 +15,8 @@
 package org.apache.geode.internal.cache;
 
 import static org.apache.geode.cache.RegionShortcut.REPLICATE;
+import static org.apache.geode.internal.lang.SystemPropertyHelper.DEFAULT_PREFIX;
```

Comment:
Why not importing the new `SystemProperty.DEFAULT_PREFIX`? Since the new `SystemProperty` class is introduced. My understanding is that the new class `SystemProperty` will be used instead of the existing `SystemPropertyHelper`.  And `SystemPropertyHelper.DEFAULT_PREFIX` is essentially `SystemProperty.DEFAULT_PREFIX`. I see there are quite a few places in this pull request that use `SystemPropertyHelper.DEFAULT_PREFIX`.

Reply:
I was trying to keep SystemPropertyHelper as a complete API such that any geode-core class that uses SystemPropertyHelper does not also need to import and use SystemProperty. We can however change the geode-core classes to use both classes.

I personally don't like SystemPropertyHelper. I think it should be deleted after moving all of the system properties into the classes that "own" those properties. Then those classes would use SystemProperty only for the dual prefixes of "geode." and "gemfire."

Reply:
I removed GEODE_PREFIX and inlined DEFAULT_PREFIX and GEMFIRE_PREFIX. Now all of the classes that referenced the prefixes via SystemPropertyHelper are referencing them from SystemProperty instead.

Reply:
Yeah, eventually it's better to have a single class for the system properties. Having two classes is confusing which class should own which properties.